src/libtiled/mapreader.cpp

```diff
@@ -401,6 +401,14 @@ void MapReaderPrivate::readTilesetTile(Tileset &tileset)
     if (!probability.isEmpty())
         tile->setProbability(probability.toFloat());
 
+    // Read tile xy-offset
+    QStringRef tileoffsetx = atts.value(QLatin1String("offsetx"));
+    QStringRef tileoffsety = atts.value(QLatin1String("offsety"));
+    if (!tileoffsetx.isEmpty())
+        tile->setOffset(QPoint(tileoffsetx.toInt(), tile->offset().y()));
+    if (!tileoffsety.isEmpty())
+        tile->setOffset(QPoint(tile->offset().x(), tileoffsety.toInt()));
```

Comment:
Since `toInt` will return 0 when the string is empty, please simplify this to:
```c++
    QPoint tileOffset(atts.value(QLatin1String("offsetx")).toInt(),
                      atts.value(QLatin1String("offsety")).toInt());
    tile->setOffset(tileOffset);
```